.github/workflows/continuous-integration.yml

```diff
@@ -10,6 +10,18 @@ jobs:
   DotNetCI:
     name: Dotnet Build & Test
     runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: ghcr.io/skyra-project/postgres:latest
+        env:
+          POSTGRES_DB: skyra-dotnet-test
```

Comment:
Only thing that puzzles me is how this is called `skyra` when the project uses `arkadia` as a default.